README.md

```diff
@@ -89,31 +89,84 @@ builds like `gcc` generate a *lot* of output), and builds timing out
 
 ## Testing conda builds locally
 
-Given a fairly empty *disposable* Ubuntu x86-64 test environment (eg,
-created with Docker, or Vagrant), it is possible to simulate *part* of
-what Travis CI will do to test building individual toolchain architectures
+Given a fairly empty *disposable* Ubuntu x86-64 test environment (e.g.,
+created with Docker or Vagrant), it is possible to simulate *part* of
+what the CI does to test building individual toolchain architectures
 locally.
 
-This can be done with something like:
+The commands from the following subsections are enough to build a package
+in a "clean" docker container based on the *ubuntu* Docker image.
 
-```
-sudo apt-get update
-sudo apt-get install wget git
+### Prerequisites
 
-# Packages from ~/.travis.yml; realpath is in coreutils in Ubuntu 18.04
-# Plus libtool and pkg-config, which are needed for openocd
-#
-# sudo apt-get install realpath
-sudo apt-get install coreutils
+The only required prerequisites are:
+* [Git](https://git-scm.com/),
+* Conda installed and initialized, e.g., using
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+(it automatically makes required *python3* with *pip* and *setuptools*
+available),
+* [conda-build-prepare](https://github.com/litex-hub/conda-build-prepare),
+* this repository cloned to a local directory.
```

Comment:
```suggestion
this repository cloned to a local directory.
```

Reply:
I'm not sure you really meant because simply placing it after the list didn't look great. I started it with `and of course this repository...` to make it more natural.